src/js/firstDiscoveryServer.js

```diff
@@ -21,10 +21,15 @@ var fdSrcDir = path.resolve(__dirname, "../../node_modules/first-discovery/src")
 
 fluid.defaults("gpii.firstDiscovery.server", {
     gradeNames: ["gpii.express"],
+    port: 8080,
```

Comment:
I changed the default to 8088. The "one stop shop" is taken care of in the index.js file. (see: https://github.com/jobara/first-discovery-server/blob/FLOE-423/index.js#L17-L19)